.github/workflows/build.yml

```diff
@@ -51,8 +51,8 @@ jobs:
       - name: Install Node dependencies
         run: npm install
 
-      - name: Run linter
-        run: npm run lint
+      # - name: Run linter
+      #   run: npm run lint
```

Comment:
This is temporary as we'll be adding this step again once we've migrated to ESLint?

Reply:
We're adding it back in the prettier PR.